benchmarks/dali_vs_python/BM_jasper/model_repository/jasper_dali/config.pbtxt

```diff
@@ -0,0 +1,46 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+name: "jasper_dali"
+backend: "dali"
+max_batch_size: 256
+input [
+  {
+    name: "DALI_INPUT_0"
+    data_type: TYPE_UINT8
+    dims: [ -1 ]
+  }
+]
+
+output [
+  {
+    name: "DALI_OUTPUT_0"
+    data_type: TYPE_FP16
+    dims: [ 64, -1 ]
+  }
+]
+
+instance_group [
+  {
+    count: 4
```

Comment:
Is it 4 instances per GPU?

Reply:
Yes. This point here is to properly measure the online scenario. Modifying the concurrency makes sense only if there are actually some additional model instances, so they can run concurrently. Plus, since we're measuring it mainly on high-memory hardware, there's no risk of having OOM problems.

Reply:
Sure. Just a note, that even a single instance makes sense with online scenario, because DALI can do dynamic batching.